### DISCUSSION TITLE
sensitivity analysis

### DISCUSSION THREAD

**Discussion by Ating24** [idaholab/moose#26834] [Q&A Tools]
When I was training a polynomial chaotic proxy model using an old version of MOOSE, I used 120 CPUs to calculate one example, but in the end, only one CPU was allocated to this example for calculation

**Comment by grmnptr** [comment]
Please update your moose, managing the number of processors per app was added in 2021 to the stochastic tools module. Originally, the batch-reset/restore options used one process per app.

**Reply by Ating24** [reply]
Is there any other way to set a single instance CPU on the old MOOSE? Does having too many polynomial chaotic trainers affect computational efficiency?

**Reply by Ating24** [reply]
For example, modifying the batch-reset/restore  ?

**Reply by Ating24** [reply]
Can we use code modification to make batch reset/restore options use multiple CPUs?
